snuba/api.py

```diff
@@ -187,13 +190,13 @@ def parse_and_run_query(validated_body, timer):
     having_conditions = body.get('having', [])
 
     aggregate_exprs = [
-        util.column_expr(col, body, alias, agg)
+        util.column_expr(dataset, col, body, alias, agg)
```

Comment:
As I got further down I see you sort of did this, but I guess there is a util version and a dataset version that it calls out to as needed.

Reply:
Yeah there is like, common functionality in condition evaluation, and then dataset-specific stuff. But, I guess it would be nicer if it wasn't split between `util` and the dataset version. Perhaps the `util` version should be in the `DataSet` base class?